### DISCUSSION TITLE
A strange error in Meteor v2.3.2 running in Android Firefox

### DISCUSSION THREAD

**Discussion by nthuTYChen** [babel/babel#13635] [Q&A]
Hi guys,

I've been developing a Meteor application and tested it in various desktop and mobile Web browsers, and it works fine in all browsers but Android Firefox. After debugging the application remotely, there seems to be an error associated to babel, as shown in the attached screenshot. The problem only occurs in Android Firefox v90.1.3 on an Android 11 device. Any suggestion? Thanks.

![firefox-bug](https://user-images.githubusercontent.com/31963114/128285728-1efc9def-574b-4a25-a3f1-30b2ba62deb0.png)

**Comment by JLHwung** [comment]
Is `@babel/runtime/helpers/createForOfIteratorerHelperLoose` in your `node_modules`? Can you share a reproduction repo?

**Reply by nthuTYChen** [reply]
Not sure if this is what you requested, but here is the list of the npm packages installed in my Meteor application. I've also check the node_module/@babel directory, but inside there is only the 'parser' directory, which seems to correspond to the @babel/parser package. Thanks.

├── @babel/parser@7.11.5
├── @types/semver@5.4.0
├── @types/underscore@1.11.2
├─┬ chalk@0.5.1
│ ├── ansi-styles@1.1.0
│ ├── escape-string-regexp@1.0.5
│ ├─┬ has-ansi@0.1.0
│ │ └── ansi-regex@0.2.1
│ ├─┬ strip-ansi@0.3.0
│ │ └── ansi-regex@0.2.1 deduped
│ └── supports-color@0.2.0
├─┬ fibers@5.0.0
│ └── detect-libc@1.0.3
├── meteor-promise@0.9.0
├─┬ multipipe@1.0.2
│ ├─┬ duplexer2@0.1.4
│ │ └─┬ readable-stream@2.3.7
│ │   ├── core-util-is@1.0.2
│ │   ├── inherits@2.0.4 deduped
│ │   ├── isarray@1.0.0
│ │   ├── process-nextick-args@2.0.1
│ │   ├── safe-buffer@5.1.2
│ │   ├── string_decoder@1.1.1 deduped
│ │   └── util-deprecate@1.0.2 deduped
│ └── object-assign@4.1.1
├─┬ node-gyp@8.0.0
│ ├── env-paths@2.2.1
│ ├─┬ glob@7.1.7
│ │ ├── fs.realpath@1.0.0
│ │ ├─┬ inflight@1.0.6
│ │ │ ├── once@1.4.0 deduped
│ │ │ └── wrappy@1.0.2
│ │ ├── inherits@2.0.4
│ │ ├─┬ minimatch@3.0.4
│ │ │ └─┬ brace-expansion@1.1.11
│ │ │   ├── balanced-match@1.0.2
│ │ │   └── concat-map@0.0.1
│ │ ├─┬ once@1.4.0
│ │ │ └── wrappy@1.0.2 deduped
│ │ └── path-is-absolute@1.0.1
│ ├── graceful-fs@4.2.8
│ ├─┬ make-fetch-happen@8.0.14
│ │ ├─┬ agentkeepalive@4.1.4
│ │ │ ├─┬ debug@4.3.2
│ │ │ │ └── ms@2.1.2 deduped
│ │ │ ├── depd@1.1.2
│ │ │ └─┬ humanize-ms@1.2.1
│ │ │   └── ms@2.1.2 deduped
│ │ ├─┬ cacache@15.2.0
│ │ │ ├─┬ @npmcli/move-file@1.1.2
│ │ │ │ ├── mkdirp@1.0.4 deduped
│ │ │ │ └── rimraf@3.0.2 deduped
│ │ │ ├── chownr@2.0.0 deduped
│ │ │ ├── fs-minipass@2.1.0 deduped
│ │ │ ├── glob@7.1.7 deduped
│ │ │ ├── infer-owner@1.0.4
│ │ │ ├── lru-cache@6.0.0 deduped
│ │ │ ├── minipass@3.1.3 deduped
│ │ │ ├── minipass-collect@1.0.2 deduped
│ │ │ ├── minipass-flush@1.0.5 deduped
│ │ │ ├── minipass-pipeline@1.2.4 deduped
│ │ │ ├── mkdirp@1.0.4 deduped
│ │ │ ├─┬ p-map@4.0.0
│ │ │ │ └─┬ aggregate-error@3.1.0
│ │ │ │   ├── clean-stack@2.2.0
│ │ │ │   └── indent-string@4.0.0
│ │ │ ├── promise-inflight@1.0.1
│ │ │ ├── rimraf@3.0.2 deduped
│ │ │ ├── ssri@8.0.1 deduped
│ │ │ ├── tar@6.1.8 deduped
│ │ │ └─┬ unique-filename@1.1.1
│ │ │   └─┬ unique-slug@2.0.2
│ │ │     └── imurmurhash@0.1.4
│ │ ├── http-cache-semantics@4.1.0
│ │ ├─┬ http-proxy-agent@4.0.1
│ │ │ ├── @tootallnate/once@1.1.2
│ │ │ ├─┬ agent-base@6.0.2
│ │ │ │ └── debug@4.3.2 deduped
│ │ │ └── debug@4.3.2 deduped
│ │ ├─┬ https-proxy-agent@5.0.0
│ │ │ ├── agent-base@6.0.2 deduped
│ │ │ └── debug@4.3.2 deduped
│ │ ├── is-lambda@1.0.1
│ │ ├─┬ lru-cache@6.0.0
│ │ │ └── yallist@4.0.0 deduped
│ │ ├─┬ minipass@3.1.3
│ │ │ └── yallist@4.0.0 deduped
│ │ ├─┬ minipass-collect@1.0.2
│ │ │ └── minipass@3.1.3 deduped
│ │ ├─┬ minipass-fetch@1.3.4
│ │ │ ├─┬ encoding@0.1.13
│ │ │ │ └─┬ iconv-lite@0.6.3
│ │ │ │   └── safer-buffer@2.1.2 deduped
│ │ │ ├── minipass@3.1.3 deduped
│ │ │ ├─┬ minipass-sized@1.0.3
│ │ │ │ └── minipass@3.1.3 deduped
│ │ │ └── minizlib@2.1.2 deduped
│ │ ├─┬ minipass-flush@1.0.5
│ │ │ └── minipass@3.1.3 deduped
│ │ ├─┬ minipass-pipeline@1.2.4
│ │ │ └── minipass@3.1.3 deduped
│ │ ├─┬ promise-retry@2.0.1
│ │ │ ├── err-code@2.0.3
│ │ │ └── retry@0.12.0
│ │ ├─┬ socks-proxy-agent@5.0.1
│ │ │ ├── agent-base@6.0.2 deduped
│ │ │ ├── debug@4.3.2 deduped
│ │ │ └─┬ socks@2.6.1
│ │ │   ├── ip@1.1.5
│ │ │   └── smart-buffer@4.2.0
│ │ └─┬ ssri@8.0.1
│ │   └── minipass@3.1.3 deduped
│ ├─┬ nopt@5.0.0
│ │ └── abbrev@1.1.1
│ ├─┬ npmlog@4.1.2
│ │ ├─┬ are-we-there-yet@1.1.5
│ │ │ ├── delegates@1.0.0
│ │ │ └── readable-stream@2.3.7 deduped
│ │ ├── console-control-strings@1.1.0
│ │ ├─┬ gauge@2.7.4
│ │ │ ├── aproba@1.2.0
│ │ │ ├── console-control-strings@1.1.0 deduped
│ │ │ ├── has-unicode@2.0.1
│ │ │ ├── object-assign@4.1.1 deduped
│ │ │ ├── signal-exit@3.0.3
│ │ │ ├─┬ string-width@1.0.2
│ │ │ │ ├── code-point-at@1.1.0
│ │ │ │ ├─┬ is-fullwidth-code-point@1.0.0
│ │ │ │ │ └── number-is-nan@1.0.1
│ │ │ │ └─┬ strip-ansi@3.0.1
│ │ │ │   └── ansi-regex@2.1.1
│ │ │ ├─┬ strip-ansi@3.0.1
│ │ │ │ └── ansi-regex@2.1.1
│ │ │ └─┬ wide-align@1.1.3
│ │ │   └── string-width@1.0.2 deduped
│ │ └── set-blocking@2.0.0
│ ├─┬ rimraf@3.0.2
│ │ └── glob@7.1.7 deduped
│ ├─┬ semver@7.3.5
│ │ └── lru-cache@6.0.0 deduped
│ ├─┬ tar@6.1.8
│ │ ├── chownr@2.0.0
│ │ ├─┬ fs-minipass@2.1.0
│ │ │ └── minipass@3.1.3 deduped
│ │ ├── minipass@3.1.3 deduped
│ │ ├─┬ minizlib@2.1.2
│ │ │ ├── minipass@3.1.3 deduped
│ │ │ └── yallist@4.0.0 deduped
│ │ ├── mkdirp@1.0.4
│ │ └── yallist@4.0.0
│ └─┬ which@2.0.2
│   └── isexe@2.0.0
├─┬ node-pre-gyp@0.15.0
│ ├── detect-libc@1.0.3 deduped
│ ├─┬ mkdirp@0.5.5
│ │ └── minimist@1.2.5
│ ├─┬ needle@2.8.0
│ │ ├─┬ debug@3.2.7
│ │ │ └── ms@2.1.2
│ │ ├─┬ iconv-lite@0.4.24
│ │ │ └── safer-buffer@2.1.2
│ │ └── sax@1.2.4
│ ├─┬ nopt@4.0.3
│ │ ├── abbrev@1.1.1 deduped
│ │ └─┬ osenv@0.1.5
│ │   ├── os-homedir@1.0.2
│ │   └── os-tmpdir@1.0.2
│ ├─┬ npm-packlist@1.4.8
│ │ ├─┬ ignore-walk@3.0.4
│ │ │ └── minimatch@3.0.4 deduped
│ │ ├─┬ npm-bundled@1.1.2
│ │ │ └── npm-normalize-package-bin@1.0.1 deduped
│ │ └── npm-normalize-package-bin@1.0.1
│ ├── npmlog@4.1.2 deduped
│ ├─┬ rc@1.2.8
│ │ ├── deep-extend@0.6.0
│ │ ├── ini@1.3.8
│ │ ├── minimist@1.2.5 deduped
│ │ └── strip-json-comments@2.0.1
│ ├─┬ rimraf@2.7.1
│ │ └── glob@7.1.7 deduped
│ ├── semver@5.4.1 deduped
│ └─┬ tar@4.4.17
│   ├── chownr@1.1.4
│   ├─┬ fs-minipass@1.2.7
│   │ └── minipass@2.9.0 deduped
│   ├─┬ minipass@2.9.0
│   │ ├── safe-buffer@5.2.1 deduped
│   │ └── yallist@3.1.1 deduped
│   ├─┬ minizlib@1.3.3
│   │ └── minipass@2.9.0 deduped
│   ├── mkdirp@0.5.5 deduped
│   ├── safe-buffer@5.2.1
│   └── yallist@3.1.1
├─┬ promise@8.1.0
│ └── asap@2.0.6
├─┬ reify@0.20.12
│ ├── acorn@6.4.2
│ ├── acorn-dynamic-import@4.0.0
│ ├─┬ magic-string@0.25.7
│ │ └── sourcemap-codec@1.4.8
│ └── semver@5.4.1 deduped
├── semver@5.4.1
├─┬ source-map-support@0.3.2
│ └─┬ source-map@0.1.32
│   └── amdefine@1.0.1
├─┬ split2@3.2.2
│ └─┬ readable-stream@3.6.0
│   ├── inherits@2.0.4 deduped
│   ├─┬ string_decoder@1.1.1
│   │ └── safe-buffer@5.1.2 deduped
│   └── util-deprecate@1.0.2
└── underscore@1.13.1
